react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -94,6 +95,13 @@
   }
 }
 
+.newBadge {
+  margin-right: (@grid-gutter-width / 4);
```

Comment:
This is probably better expressed as `1ch` or factor of. Given we want it to be closer to a space rather than relate to the gutter width. If you are concerned about below Safari 7 i'd suggest `1ex` as a fallback. Scaling to factors of these units will keep the spacing proportionate to the text which i think is the goal